src/System.Net.Http/src/System/Net/Http/Headers/HeaderUtilities.cs

```diff
@@ -65,21 +65,27 @@ internal static void SetQuality(ObjectCollection<NameValueHeaderValue> parameter
             }
         }
 
-        // Encode a string using RFC 5987 encoding.
-        // encoding'lang'PercentEncodedSpecials
-        internal static string Encode5987(string input)
+        // tspecials are required to be in a quoted string.  Only non-ascii needs to be encoded.
```

Comment:
This comment seems out of place.  And the method can just be named "ContainsNonAscii", since that's what it's checking.